components/test_raftstore/src/transport_simulate.rs

```diff
@@ -753,3 +753,23 @@ impl Filter for DropMessageFilter {
         Ok(())
     }
 }
+
+#[derive(Clone)]
+pub struct MessageCorruptFilter<F: Fn(&mut RaftMessage) + Clone> {
```

Comment:
Prefer to `MessageCorruptFilter<F: Fn(&mut RaftMessage) + Clone>(F)` and remove the `new` method.